tests/integration/seccomp-notify.bats

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	teardown_seccompagent
+	setup_seccompagent
+
+	teardown_busybox
+	setup_busybox
+}
+
+function teardown() {
+	teardown_seccompagent
+	teardown_busybox
+}
+
+@test "runc run (seccomp notify tests)" {
+	requires root
+	requires no_systemd
+
+	if [ "$KERNEL_MAJOR" -lt 5 ]; then
+		skip "requires kernel 5.6"
+	elif [ "$KERNEL_MINOR" -lt 6 ]; then
+		skip "requires kernel 5.6"
```

Comment:
If we have kernel `6.1` (in the far future), this would erroneously print `requires kernel 5.6`

tests/integration/cgroups.bats has an example how to test `KERNEL_MINOR` correctly.

Reply:
Maybe with a comment:
```
# to fetch the seccomp fd, runc uses the pidfd_getfd system call, available in Linux >= 5.6
# https://github.com/torvalds/linux/commit/8649c322f75c96e7ced2fec201e123b2b073bf09
```

Reply:
ops, I know there was something weird in this test. Thanks for pointing it out.